ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/optimizer.yml

```diff
@@ -0,0 +1,5 @@
+---
+# Default application spec for Optimizer
+mas_app_ws_spec:
+  bindings:
+    manage: workspace
```

Comment:
Optimizer shouldn't have this binding ... it's not valid to specify app to app bindings here, see dioscussion in Slack from last week where I believe it was confirmed this is not even needed.

Reply:
Waiting for @lee-cotton 's double confirmation here: https://ibm-watson-iot.slack.com/archives/C02AF89FYF9/p1653427785449579?thread_ts=1653413356.140309&cid=C02AF89FYF9 (confusion between Manage and Workspace binding, not sure if it was meant to be the same behind the scenes)

Reply:
Manage code is not already ready to remove it: https://ibm-watson-iot.slack.com/archives/C02AF89FYF9/p1653461536070859?thread_ts=1653413356.140309&cid=C02AF89FYF9